src/test/java/resources/bulkPayment.feature

```diff
@@ -168,3 +339,258 @@ Feature: Test ability to make payment to individual with bank account
+    #payee 6 creation
+    When I create and setup a "payee" with id "6" and account balance of 20 for all combine test cases
+    Given I have tenant as "payeefsp3"
+    Then I call the balance api for payee "6" balance for all combine test cases
+
+    Then add row to csv with current payer and payee, payment mode as "gsma" and transfer amount 6 and id 5 for all combine test cases
+
+        #payer 7 creation
+    Given I have Fineract-Platform-TenantId as "payerfsp2"
+    When I create and setup a "payer" with id "7" and account balance of 50 for all combine test cases
+    Given I have tenant as "payerfsp"
+    Then I call the balance api for payer "7" balance for all combine test cases
+    #payee 7 creation
+    When I create and setup a "payee" with id "7" and account balance of 20 for all combine test cases
+    Given I have tenant as "payeefsp3"
+    Then I call the balance api for payee "7" balance for all combine test cases
+
+    Then add row to csv with current payer and payee, payment mode as "gsma" and transfer amount 7 and id 6 for all combine test cases
+    #payer 8 creation
+    Given I have Fineract-Platform-TenantId as "payerfsp2"
+    When I create and setup a "payer" with id "8" and account balance of 30 for all combine test cases
+    Given I have tenant as "payerfsp"
+    Then I call the balance api for payer "8" balance for all combine test cases
+    #payee 8 creation
+    When I create and setup a "payee" with id "8" and account balance of 30 for all combine test cases
+    Given I have tenant as "payeefsp3"
+    Then I call the balance api for payee "8" balance for all combine test cases
+
+    Then add row to csv with current payer and payee, payment mode as "gsma" and transfer amount 8 and id 7 for all combine test cases
+
+            #payer 9 creation
+    Given I have Fineract-Platform-TenantId as "payerfsp2"
+    When I create and setup a "payer" with id "9" and account balance of 50 for all combine test cases
+    Given I have tenant as "payerfsp"
+    Then I call the balance api for payer "9" balance for all combine test cases
+    #payee 9 creation
+    When I create and setup a "payee" with id "9" and account balance of 20 for all combine test cases
+    Given I have tenant as "payeefsp3"
+    Then I call the balance api for payee "9" balance for all combine test cases
+
+    Then add row to csv with current payer and payee, payment mode as "gsma" and transfer amount 9 and id 8 for all combine test cases
+    #payer 10 creation
+    Given I have Fineract-Platform-TenantId as "payerfsp2"
+    When I create and setup a "payer" with id "10" and account balance of 30 for all combine test cases
+    Given I have tenant as "payerfsp"
+    Then I call the balance api for payer "10" balance for all combine test cases
+    #payee 10 creation
+    When I create and setup a "payee" with id "10" and account balance of 30 for all combine test cases
```

Comment:
check if examples or data tables can be used to reduce the number of steps effectively